doc/user-guides/secure-protect-connect.md

```diff
@@ -395,7 +443,7 @@ The gateway limits are a good set of limits for the general case, but as the dev
 
 ```sh
 kubectl --context $KUBECTL_CONTEXT apply -f - <<EOF
-apiVersion: kuadrant.io/v1beta3
```

Comment:
For `main` this change does not seem correct

Reply:
fixed